libs/application/templates/institution-application/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Since we are already in the application folder I would name the library `institution` without the application as we do for the other templates.